### PR TITLE
Remove RSG from the components

### PR DESCRIPTION
**Pull request by radi-cho**
I removed `"RSG"` from the components.
@retrixe - please review, @FriendsNone - take a look...
When merge this we can publish beta version of `3.0.0`, because the most important bugs and features are added.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,41 +4,41 @@
 
 ### 2.0.0
 
-[The component `RSGButton` has been introduced.](https://github.com/RSG-Group/rsg-components/wiki/RSGButton)
```
Comment (retrixe) [inline CHANGELOG.md]: Don't edit the CHANGELOG -.-

File: example/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # example of rsg-components
-Click a button to send data from the server, fetch the same data from the server and show it in an RSGBox component. Powered by Meteor and uses the development version of rsg-components (aka you must build rsg-components before using this). Example of usage of all components in RSGComponents (WIP).
```
Comment (retrixe) [inline example/README.md]: Rollback changes to this as well.

Comment (retrixe) [reply example/README.md]: Well, the ones which mention RSGComponents. Not FormBasic.

File: CHANGELOG.md (hunk 1 above)
Comment (retrixe) [reply CHANGELOG.md]: The 3.0.0 change log will note these changes.

**Comment by retrixe** [comment]
Can we have a fix on the issues please?

**Comment by radi-cho** [comment]
If you have time do it. Otherwise I will do it after an hour, two...

File: example/README.md (hunk 2 above)
Comment (retrixe) [reply example/README.md]: This has been addressed.

**Pull request merged**